File: 0441-arranging-coins/0441-arranging-coins.js
/**
 * @param {number} n
 * @return {number}
 */
var arrangeCoins = function(n) {
    let sCount = 0
    let stairs = 1
    while (n >= 0){
         n = n - stairs;
        if ( n >= 0){
          
            stairs++
            sCount++
        }
    }
    return sCount
};

File: 0441-arranging-coins/0441-arranging-coins.py
class Solution(object):
    def arrangeCoins(self, n):
        """
        :type n: int
        :rtype: int
        """
        sCount = 0
        stairs = 1
        while n >= 0:
            n = n - stairs
            if n >= 0:
                sCount += 1
                stairs += 1
            
        return sCount